package.json

```diff
@@ -0,0 +1,54 @@
+{
+    "name": "event-maker-telegram-bot",
+    "version": "1.0.0",
+    "description": "",
+    "main": "dist/index.js",
+    "scripts": {
+        "prebuild": "rimraf dist",
+        "build": "rollup --config",
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "format": "prettier --ignore-path .gitignore \"**/*.+(js|json|ts|md)\"",
+        "format:check": "npm run format -- --list-different",
+        "format:write": "npm run format -- --write",
+        "types:check": "tsc",
+        "lint": "eslint --ignore-path .gitignore --ext .js,.ts .",
+        "validate": "npm run types:check && npm run format:check && npm run lint && npm run build"
```

Comment:
1. do you think is it good idea to use this command in #2 ?
2. why to have `npm run build` here?

Reply:
1. no, instead of `format:check` I'd rather use `format:write` in CI. And in CI I'd use `npm run build` separately, so it's more explicit and clear in CI.
2. This command is more for self-check or for a precommit hook, so we know that it's buildable before spending CI resources.